tests/commentsHandler/test.ts

```diff
@@ -58,18 +67,28 @@ describe('comments', () => {
   });
 
   it('should return comments by property id', async () => {
-    const data = await propertyInsert()(createRequestFromBlueprint({
+    const request1 = createRequestFromBlueprint({
       name: 'name',
       description: 'description',
-    }));
+    });
 
-    const propertyId = JSON.parse(data.body).id;
+    const data1 = await propertyInsert()(request1);
 
-    const request = createRequestFromBlueprint({}, {
+    const propertyId = JSON.parse(data1.body).id;
+
+    const request2 = createRequestFromBlueprint({
+      text: 'Horrible hotel, never will stay here again',
+    }, {
       id: propertyId,
     });
 
-    const result = await commentsRoutes.getCommentsByPropertyId()(request);
+    await commentsRoutes.createPropertyComment()(request2);
+
+    const request3 = createRequestFromBlueprint({}, {
+      id: propertyId,
+    });
+
+    const result = await commentsRoutes.getCommentsByPropertyId()(request3);
 
     expect(result.statusCode).to.be.equal(200);
     expect(JSON.parse(result.body)).to.be.an('array');
```

Comment:
maybe let's assert some properties too? So that if anyone breaks the format, the unit test will fail